### ISSUE TITLE
Not working on Windows

### ISSUE THREAD

**Issue by phoenicyan** [cdeguzman95/AABot#1]
I tried to launch bot.py on Windows (python 3.6), but the following error was returned:
PS F:\Src\AABot-master> python .\bot.py
Traceback (most recent call last):
  File ".\bot.py", line 1, in <module>
    from airport_info.airports import *
  File "F:\Src\AABot-master\airport_info\airports.py", line 13, in <module>
    for row in reader:
  File "C:\Python36\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x8d in position 2098: character maps to <undefined>

Any ideas?

**Comment by cdeguzman95** [comment]
You might need to specify the encoding of the file; I wrote this in Linux so it likely defaulted to UTF-8 on my side. In line 11 of airports.py try this:

`with open(file_path, newline='', encoding='utf-8') as csvfile:`

**Comment by phoenicyan** [comment]
Thank you! It fixed the issue.

I had to execute the following commands as well:
pip3 install requests
pip3 install telegram

But then got the following error:
 File ".\bot.py", line 5, in <module>
   from telegram import ReplyKeyboardMarkup, ReplyKeyboardRemove
ImportError: cannot import name 'ReplyKeyboardMarkup'

Should a different version of Telegram be installed in Python3?

**Comment by cdeguzman95** [comment]
`pip install -r requirements.txt` should get you all of the required libraries to run the bot; the Telegram wrapper I use in particular is called 'python-telegram-bot'.

**Comment by phoenicyan** [comment]
Yes, installation of the requirements.txt helped! Thank you!

I also created a file named "token" in the bot.py folder, and placed some token in there (it looks like 123456789:blahblehbaz). Then I ran bot.py, but nothing is happening...

I have Telegram Desktop running locally, but it does not get notifications or anything... I expected the AA bot somehow appear in the Telegram. How can I troubleshoot?

**Comment by cdeguzman95** [comment]
It would probably be best for you to create a new bot with Telegram so that you can get your own API key; [information on doing so can be found here](https://core.telegram.org/bots#6-botfather). Once you get an API key, store that in the token file and then run it, making sure to message the bot you just created.

**Comment by phoenicyan** [comment]
Finally, I got back to playing with the bot today. As you suggested, I created brand new bot and user, copied token text, and launched the bot.py. Then inside Telegram Desktop I went to the bot link and clicked START button. Then list of commands was shown, and they worked!
Later I copied token of an old existing bot, and typed "/start", and the AAbot worked!

Thank you for publishing the tool! I think that there are several things that can be added or improved.
